doc/material-controls-extensions.md

```diff
@@ -9,7 +9,7 @@ Below is a summary of the icon support for different controls:
 | **Button**      |  ✔️  |     ❌     |      ❌      |
 | **Combobox**    |  ✔️  |     ❌     |      ❌      |
 | **PasswordBox** |  ✔️  |     ❌     |      ❌      |
-| **TextBox**     |  ❌  |     ✔️     |      ✔️      |
+| **TextBox**     |  ✔️  |     ✔️     |      ✔️      |

```

Comment:
add a `\*` with a foot note explaining that it will just set the LeadingIcon instead